3.feature_extraction/README.md

```diff
@@ -0,0 +1,18 @@
+# Perform segmentation and extract morphology features
+
+In this module, we use CellProfiler to segment nuclei, cells, cytoplasm, and organoids, also known as compartments.
+We then extract morphology features (e.g., texture, granularity, area, etc.) using the compartments as masks across all channels.
+We also extract some whole image features, but these are not prioritized in downstream analysis.
```

Comment:
what do you mean "prioritized"? Do you actually mean not considered?

FWIW, in this work, whole image might actually be interesting.... oh, but wait, you're performing whole organoid feature extraction so it's probably very similar

Reply:
Maybe we can say whole image features are not included in downstream analysis which is true for all projects, plus I agree whole organoid would probably be very similar to whole images anyway.